packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -2831,6 +2832,16 @@ const schema: SchemaType<"Posts"> = {
     canCreate: ['admins'],
     canUpdate: [userOwns, 'admins'],
   },
+
+  lastRevisionAt: resolverOnlyField({
+    type: Date,
+    optional: true,
+    canRead: ['guests'],
+    resolver: async (post: DbPost, args: void, context: ResolverContext) => {
+      const lastRevision = await Revisions.findOne({documentId: post._id}, {sort: {editedAt: -1}});
```

Comment:
This should use `context.Revisions` to avoid importing the `Revisions` collection into the file